frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -25,6 +21,11 @@
           .c-icons
             i.icon-chevron-right
 
+        hr.tab-nav-separator
+
+      .tab-nav-list.is-subtitle Version 1.6.0
+      .tab-nav-list.tab-nav-ack.is-subtitle Acknowledgements
```

Comment:
1. Missing `i18n`
2. Doesn't actually doing anything. I would comment out for now with a `TODO` that points to an issue

Reply:
Make sense.

Reply:
That issue exists btw: https://github.com/okTurtles/group-income/issues/673